app/views/content_items/topic.html.erb

```diff
@@ -30,44 +30,58 @@
-              <h2 class="govuk-heading-m govuk-!-margin-bottom-1 govuk-!-margin-top-3 subsection__title" id="<%= link_group.name.parameterize %>"><%= link_group.name %></h2>
-            <% if link_group.description.present? %>
-              <p class="govuk-body govuk-!-margin-bottom-1 subsection__description"><%= link_group.description %></p>
-            <% end %>
+              <%= render "govuk_publishing_components/components/heading", {
+                text: link_group.name,
+                font_size: "m",
+                margin_bottom: 1,
+                id: link_group.name.parameterize,
+              } %>
+              <% if link_group.description.present? %>
+                <p class="govuk-body govuk-!-margin-bottom-1 subsection__description"><%= link_group.description %></p>
+              <% end %>
             </div>
           <% end %>
-          <ul class="govuk-list govuk-!-margin-bottom-9">
-            <% link_group.linked_items.each { |linked_item| %>
-              <li>
-                <%= link_to linked_item.label, linked_item.href, class: 'govuk-link' %>
-              </li>
-            <% } %>
-          </ul>
-        <% } %>
+          <%
+            link_items = []
+
+            link_group.linked_items.each do |linked_item|
+              link_items << sanitize(link_to linked_item.label, linked_item.href, class: 'govuk-link')
+            end
+          %>
+          <%= render "govuk_publishing_components/components/list", {
+            items: link_items
+          } %>
+        <% end %>
       <% end %>
     </div>
 
     <div class="govuk-grid-column-one-third">
       <aside class="related govuk-!-margin-top-3">
       <% if @content_item.content_owners.any? %>
         <div class="related-item govuk-!-padding-top-4 govuk-!-margin-bottom-5">
-          <h2 class="related-item__title govuk-heading-s govuk-!-margin-bottom-2" id="related-communities">
-            <%= topic_related_communities_title(@content_item.content_owners) -%>
-          </h2>
+          <%= render "govuk_publishing_components/components/heading", {
+            text: topic_related_communities_title(@content_item.content_owners),
+            font_size: "s",
+            margin_bottom: 2,
+            id: "related-communities",
+          } %>
           <p class="related-item__description govuk-body">
             Find out what the cross-government community does and how to get involved.
           </p>
           <nav role="navigation" aria-labelledby="related-communities" class="related-communities">
-            <ul class="related-item__list govuk-list">
-              <% @content_item.content_owners.each do |content_owner| %>
-                <li class="related-item__list-item govuk-body-s">
-                  <%= link_to content_owner.title, content_owner.href, class: 'govuk-link' %>
-                </li>
-              <% end %>
-            </ul>
+            <%
+              related_items = []
```

Comment:
I'm seeing this pattern on some of the stuff i've been doing as well. Might be a case to extend the gem in some way.

Reply:
When you say this pattern what do you mean?

Also I've just realised from looking at this again that this could totally be mapped. Will update.